src/ui-kit/Pagination/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { setCurrentPage } from 'redux/actions'
+import { Pagination } from 'antd'
+import './pagination.scss'
+
+const PaginationKit = ({setCurrentPage, pageCurrent}) => {
+
+  const setCurrentPageHandler = (page) => {
+    setCurrentPage(page)
+  }
+
+  return (
+    <div>
+      <Pagination
+        simple
+        className='ui-kit__pagination'
+        onChange={setCurrentPageHandler}
+        defaultCurrent={pageCurrent}
+        total={500}
+      />
+
+    </div>
+  )
+}
+
+const mapDispatchToProps = {
+  setCurrentPage
+}
```

Comment:
mapDispatchToProps should not be in this component, please move it outside in the parent component so you can receive props here